dpclean/dp/dp_select_samples.py

```diff
@@ -24,10 +24,10 @@ def evaluate(self,
         e, f, v = self.dp.eval(coord, cell, atype)
         return e[0], f[0], v[0].reshape([3, 3])
 
-    def validate(self, systems, train_params, batch_size="auto", optional_args=None):
+    def validate(self, systems, train_params, batch_size="auto", optional_args=None, backend="pt"):
         with open("valid.txt", "w") as f:
             f.write("\n".join([str(sys) for sys in systems]))
-        cmd = "dp test -m %s -f valid.txt -n 99999999 -d result" % self.model
+        cmd = "dp --%s test -m %s -f valid.txt -n 99999999 -d result" % (backend, self.model)
```

Comment:
Backend can be inferred from the suffix of the model file.

Reply:
Does that mean if the suffix of model is "pt", command `dp test` also works, so that {backend} is not necessary here?